app/controllers/api/v1/images_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Api
+  module V1
+    class ImagesController < ApplicationController
+      before_action :set_recipe
+
+      def create
+        service = ImagesService.new(image_params, @recipe)
+        if service.create
+          render status: :ok
```

Comment:
```suggestion
          render status: :created
```

cuz ok returns `200` but you should return here `201`

Reply:
Fixed.